zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
```

Comment:
Lets just do this:

    def pick(choices, message, allow_none=False):
        """Interactively pick from a list of possibilities."""
        if not choices:
            return None
        if allow_none:
            choices = [None] + choices
        if len(choices) > 1:
            click.clear()
            inquirer_type = inquirer.Checkbox if allow_multiple else inquirer.List
            questions = [inquirer_type(' ', message=message, choices=choices)]
            response = inquirer.prompt(questions)
            if response is None:
                raise KeyboardInterrupt
            return response[' ']
        return choices[0]
    
    
    def pick_multiple(choices, message):
        """Interactively pick multiple items from a list of possibilities."""
        if not choices:
            return None
        click.clear()
        response = inquirer.prompt([inquirer.Checkbox(' ', message=message, choices=choices)])
        if response is None:
            raise KeyboardInterrupt
        return response[' ']